src/fidesops/service/connectors/query_config.py

```diff
@@ -735,7 +735,12 @@ def generate_query(
                 path = path.replace(f"<{param.name}>", input_data[param.name][0])
 
         logger.info(f"Populated request params for {current_request.path}")
-        return "GET", path, params, None
+        method: HTTPMethod = (
+            current_request.method.value
+            if current_request.method
+            else HTTPMethod.GET.value
+        )
+        return method, path, params, None
```

Comment:
It would be useful here if we could explicitly set `body = None` earlier in the method, then return `body` here. Having the var there keeps it readable without going away to lookup what `SaaSRequestParams` is.

Can we also make `SaaSRequestParams` a schema in its own right?

Reply:
Hey @seanpreston, I was thinking the same thing about `SaaSRequestParams` it's getting a bit clunky in my use cases too. Can we ticket this as a separate enhancement to not hold up the custom HTTP methods?